Contents/Items/Card/Card.cs

```diff
@@ -576,9 +577,9 @@ class PlayerCardHandle : ModPlayer {
 	}
 	class CardNPCdrop : GlobalNPC {
 		public override void ModifyNPCLoot(NPC npc, NPCLoot npcLoot) {
-			LeadingConditionRule NightmareMode = new LeadingConditionRule(new NightmareMode());
-			NightmareMode.OnSuccess(ItemDropRule.ByCondition(new Conditions.LegacyHack_IsABoss(), ModContent.ItemType<PremiumCardPacket>()));
-			npcLoot.Add(NightmareMode);
+			LeadingConditionRule HardcoreMode = new LeadingConditionRule(new HardcoreExclusive());
+			HardcoreMode.OnSuccess(ItemDropRule.BossBagByCondition(new HardcoreExclusive(), ModContent.ItemType<PremiumCardPacket>()));
+			npcLoot.Add(HardcoreMode);
```

Comment:
Can't this be simplified to just
`npcLoot.Add(ItemDropRule.BossBagByCondition(new HardcoreExclusive(), ModContent.ItemType<PremiumCardPacket>()));`

Reply:
oh I didn't notice that I can do that, will do it now